core/src/main/scala/spinal/core/core.scala

```diff
@@ -76,6 +76,8 @@ package object core extends BaseTypeFactory with BaseTypeCast {
 
     def downto(start: Int): Range.Inclusive = Range.inclusive(i, start, -1)
 
+    def ~(start: Int): Range.Inclusive = this.downto(start)
```

Comment:
do you mind add `~` for short of `downto`
```scala
a(8 downto 4)
a(8 ~ 4) 
```

Reply:
Issue with ~ is that it is already a integer operator (bitwise not), so that's quite confusing

Reply:
` ~4`(bitwise not)  is a UnaryOperator, `8 ~ 4` is a BinaryOperator.  not perfect， But there seems to be no better choice in `~!@#$%^&*` :joy:
for some scenarios `downto` is too long 
```scala
ret := B(0,6 bits) ## ab(2 downto 0) ## ab(5 downto 3) ## ab(8 downto 6) ## ab(11 downto 9)} 
```
```scala
ret := B(0,6 bits) ## ab(2 ~ 0) ## ab(5 ~ 3) ## ab(8 ~ 6) ## ab(11 ~ 9)
```

Reply:
I am also not in favor of adding this shorthand. You can and most probably should split lines once they get too long or just for readability, and `~` won’t change that problem.

Reply:
ok, i will remove that